### DISCUSSION TITLE
有人投毒了

### DISCUSSION THREAD

**Discussion by galaxy-sea** [getlantern/lantern-forum-cn#854] [General]
从 https://github.com/getlantern/lantern 页面下载 https://gitlab.com/getlantern/lantern-binaries-mirror/-/raw/master/lantern-installer.dmg 进行安装结果是一个中国公司收款方

![IMG_3697](https://github.com/user-attachments/assets/e23f9d09-cd96-4849-8803-8cd840de5787)
![IMG_3696](https://github.com/user-attachments/assets/91684960-ae35-4dfd-906b-709bfcbd44db)

**Comment by galaxy-sea** [comment]
hello @Mike777Khoo  , can you help me solve my payment issue? I don’t know why my payment method is linked to a phishing platform.

**Reply by MaxMKMax** [reply]
@galaxy-sea Okay sure, let me check with the team and see what happened. I will let you know. But if the payment has been made to the third party out of Lantern, we could not help much. But i will help you to check with the team and get back to you see if any solutions.

**Reply by greySkiesMarsh** [reply]
@galaxy-sea Hello, I'm looking into your issue now. If you could, please send me an email at marsh@getlantern.org using the email that you use for Lantern Pro, and I can generate a unique and secure payment link for you.

**Comment by MaxMKMax** [comment]
@galaxy-sea 我帮您问着了. 您要小心假冒的平台, 别再被骗了. 也要小心后续的问题.

**Comment by MaxMKMax** [comment]
@galaxy-sea 您好. 刚才和您确认过了. 那个支付系统是我们第三方的. 应该不是假冒的. 

请问您的帐号遇到什么问题吗? 我这里帮您提交问题上去解决. 但需要您的账号邮箱和支付单据.

**Reply by galaxy-sea** [reply]
为什么突然从官方收款换成第三方收款啊

**Comment by Honeydukes3** [comment]
我就遇到了同样的情况，付费账号是771324905donald@tutanota.com
![1](https://github.com/user-attachments/assets/b62473ba-337d-4d64-8b88-c478ab6b77aa)
@Mike777Khoo

**Reply by Honeydukes3** [reply]
如果看到请回复一下，谢谢！

**Reply by galaxy-sea** [reply]
@Honeydukes3 狆國亾芣騙狆國亾，伎苻寶舉蕔姟账啴，伎苻寶浍幇沵蹆窾哋
